cmd/minikube/cmd/status.go

```diff
@@ -136,6 +137,8 @@ type Status struct {
 	Kubeconfig string
 	Worker     bool
 	TimeToStop string
```

Comment:
Would u plz paste the output of minikube status —outpus=json after this PR ?
Does this field reflect in the json output ?

Reply:
@medyagh, I added the output for `minikube status —outpus=json` in the pr description